### PR TITLE
fix: reduced the height of the chart wrapper that is making the butto…

### LINKED ISSUE #2610: Bug: Show latest PRs is broken
### Describe the bug

Screen Shot 2024-02-07 at 1 18 59 AM

When viewing the the contributors tab and looking at the contributors list, I can't click to see the list of PRs.

### Steps to reproduce

1. Visit oss.fyi/aigrant
2. Click on the contributors tab
3. Note it doesn't working when clicking show more pull requests. 

### Browsers

_No response_

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by takanome-dev**
## Description

This pr reduced the height of the EChartWrapper wrapper that is making the `show latest prs` button unclikable and the repo list tooltips not working

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #2610
## Mobile & Desktop Screenshots/Recordings

https://github.com/open-sauced/app/assets/79809121/59fa494e-de87-4b32-95a9-c566515dea9e

## Steps to QA

1. go to the explore page
2. click on the contributors tab
3. change the contributor's view
4. click on the button `show latest prs` and hover over the repo list

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/organisms/ContributorCard/contributor-card.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ const ContributorCard = ({ className, contributor, topic, repositories, range }:
           </div>
         </div>
         <div className="h-32">
-          <CardLineChart lineChartOption={chart} />
+          <CardLineChart lineChartOption={chart} className="!h-36" />
```
Comment (nickytonline) [inline components/organisms/ContributorCard/contributor-card.tsx:61]: refactor (blocking): There's no need to use `!important`. Set a max height instead.

```suggestion
          <CardLineChart lineChartOption={chart} className="max-h-36" />
```

File: components/organisms/ContributorCard/contributor-card.tsx [hunk 2]
```diff
@@ -69,7 +69,7 @@ const ContributorCard = ({ className, contributor, topic, repositories, range }:
         <div className="flex justify-center w-full">
           <button
             onClick={() => setShowPRs((prevState) => !prevState)}
-            className="w-full py-1 transition bg-white border rounded-lg border-light-slate-6 hover:bg-light-slate-1"
+            className="w-full py-1 transition bg-white border rounded-lg border-light-slate-6 hover:bg-light-slate-3"
```
Comment (nickytonline) [inline components/organisms/ContributorCard/contributor-card.tsx:72]: These changes aren't related to the fix. Please open a separate issue to discuss these changes.

```suggestion
            className="w-full py-1 transition bg-white border rounded-lg border-light-slate-6 hover:bg-light-slate-1"
```

**Review by nickytonline — CHANGES_REQUESTED** [review]
Thanks for the fix @takanome-dev! A couple change requests.

**Comment by takanome-dev** [comment]
Thanks @nickytonline :+1:

**Review by nickytonline — APPROVED** [review]
Thanks @takanome-dev! :shipit:

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**